src/app/self-service/users/user.service.ts

```diff
@@ -64,7 +64,7 @@ export class UserService {
    */
   changePassword(userId: string, passwordObj: any) {
 
-    return this.http.put(`/self/user/${userId}`, passwordObj);
+    return this.http.put(`/users/${userId}`, passwordObj);
```

Comment:
@IOhacker Are you sure we should call the /users API from self service? I think this is wrong. There is an Self service user API, which can be called:
`PUT /self/user`

Reply:
@adamsaghy I think that if we are going to use the Self Service for changing the password of the User, then the we should use the Self Service Authentication for previously get the Access Token.

Reply:
I dont really know what the access token to do here... Behind the curtain  kind of the same happens, either API is called. My only concern here: If this is a UI for self service, I think the Self service API should be called.

There is UI view for (staff) user and for self-service. Kind of the same at backend side: there is API for self service and the regular API

Reply:
However, if you say this is not self-service and this is something really different, I am kind of okay with that as well. Like I said, kind of the same happens, either API is called